test/ts/unit/collateralized/collateralized_contract.ts

```diff
@@ -10,9 +10,8 @@ import {
     CollateralSeized,
 } from "../../logs/collateralized_contract";
 
-import {
-    MockCollateralizedTermsContractContract,
-} from "../../../../types/generated/mock_collateralized_terms_contract";
+// tslint:disable-next-line
+import { MockCollateralizedTermsContractContract } from "../../../../types/generated/mock_collateralized_terms_contract";
```

Comment:
is it possible to access these modules more simply?

Something like: `"types/generated"`